pythainlp/transliterate/core.py

```diff
@@ -9,22 +9,24 @@
 
 
 def romanize(
-    text: str,
+    word: str,
```

Comment:
This may break some code that call the romanize function by `romanize(text="...")`

Reply:
Fixed